src/main/java/decodes/decoder/Nos6Min.java

```diff
@@ -3,6 +3,15 @@
 *
 * $Log$
 *
+* 2023/08/28 Baoyu Yin
+* added checking for primary WL sensor with wrong raw data (@@@ and ???) - ING697
+*
```

Comment:
No need to change anything now, but these types of header blocks get unwieldy quickly. Stick this information in the commit message in the future.